### ISSUE TITLE
Ships do not save out any built in reverse/steering flares sprites/sounds

### ISSUE THREAD

**Issue by Dogmeat137** [endless-sky/endless-sky#5476]
**Describe the bug**
Reverse engine flares disappear after reloading a save

**To Reproduce**
1. Acquire the Penguin or any ship with reverse engine flares
2. Fly backwards and notice the reverse flares
3. Reload the save and Depart
4. Reverse flares no longer work

**Expected behavior**
Reverse flares work when reverse thrusting

**System (please complete the following information):**
 - OS: Windows 10 Pro, version 1903 build 18362.1139
 - Game Source: Github
 - Version: 0.9.13 Continuous 97afe58

**Link to save file**

WORLD FORGE

https://hastebin.com/ucicifesab.yaml

**Comment by Amazinite** [comment]
Looks like the problem isn't with reverse flares themselves, but with the fact that the reverse flares are defined by the ship, while the steering and thrusting flares are defined by the engines.
```
ship "Penguin"
		...
	attributes
		...
		"reverse thrust" 16
		"reverse thrusting energy" 2.5
		"reverse thrusting heat" 2.0
		# Reverse flare sprite and sound defined here.
		"reverse flare sprite" "effect/remnant flare/small"
			"frame rate" 5
		"reverse flare sound" "plasma tiny"
	outfits
		...
		# Steering and thrusting flare sprites and sounds defined by these outfits.
		"Crucible-Class Thruster"
		"Crucible-Class Steering"
		...
```
But here's a Penguin that has been written to the save file.
```
ship Penguin
		...
	attributes
		...
		"reverse thrust" 16
		"reverse thrusting energy" 2.5
		"reverse thrusting heat" 2
		# "reverse flare sprite" and "reverse flare sound" aren't saved.
		...
	outfits
		# Outfits obviously still present.
		"Crucible-Class Steering"
		"Crucible-Class Thruster"
		...
```
Looks to be an oversight on my part when I first added reverse/steering flares. These attributes don't simply have numerical values like other attributes, so they need to be saved separately, which I hadn't done.
https://github.com/endless-sky/endless-sky/blob/721909cb680a45a7552f9aa0319447eac9126ce4/source/Ship.cpp#L686-L705
I imagine that the hyper/jump sound and effect attributes need to be saved here as well.

**Comment by Amazinite** [comment]
Fixed by 17dcc29.
